pom.xml

```diff
@@ -18,10 +18,10 @@
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus-plugin.version>1.8.1.Final</quarkus-plugin.version>
+        <quarkus-plugin.version>1.12.2.Final</quarkus-plugin.version>
```

Comment:
This branch needs to be rebased onto `main` branch and once done the version will be the last one

Reply:
done